### PR TITLE
AWS Tutorials Cleanup

### LINKED ISSUE #1640: Clarify current directory in userData
In your tutorials, such as https://www.pulumi.com/docs/tutorials/aws/ec2-webserver/, please clarify that `userData` is running in the root directory so files being written end up there (instead of the home directory). It took me a long time to figure this out :)

### PR DESCRIPTION
**Pull request by bermudezmt**
Update cleanup shortcode
Update all AWS featured tutorials
Update webserver tutorial to use shortcode
Fix #1640 

Signed-off-by: bermudezmt <bermudez.mt@gmail.com>

### Related issues (optional)

Partially addresses #1671 and closes #1640

### CODE REVIEW

File: content/docs/tutorials/aws/ecs-fargate.md [hunk 1]
```diff
@@ -1,149 +1,194 @@
...
 
 aliases: ["/docs/reference/tutorials/aws/tutorial-service/"]
 ---
 
-In this tutorial, we'll build and publish a Docker container image to a private Elastic Container Registry (ECR), and
-spin up a load balanced Amazon Elastic Container Service (ECS) "Fargate" service, all in a handful of lines of code,
-using [Pulumi Crosswalk for AWS]({{< relref "/docs/guides/crosswalk/aws" >}}).
+<!--To-Do: Update github-buttons shortcode to check for existence of examples-->
+<p class="mb-4 flex">
+    <button class="flex flex-wrap items-center rounded text-xs text-white bg-blue-600 border-2 border-blue-600 px-2 mr-2 whitespace-no-wrap hover:text-white h-8" href="https://github.com/pulumi/examples/tree/master/aws-ts-containers" target="_blank">
```
Comment (justinvp) [inline content/docs/tutorials/aws/ecs-fargate.md]: Clicking the button doesn't do anything. It needs to be an `<a>` not a `<button>`.

File: content/docs/tutorials/aws/ecs-fargate.md [hunk 2]
```diff
@@ -1,149 +1,194 @@
...
 
-## Deploying the App and Infrastructure
+## Deploy the App
+
+### Step 1: Create a new project from a template
 
-To get started, we will create a project, initialize it, then run `pulumi up`:
+Create a project directory, `hello-fargate`, and change into it. Run [`pulumi new aws-typescript --name myproject`]({{< relref "/docs/reference/cli/pulumi_new" >}}) to create a new project using the AWS template for TypeScript. Replace `myproject` with your desired project name.
 
 1.  Run `pulumi new` to create a new project:
```
Comment (justinvp) [inline content/docs/tutorials/aws/ecs-fargate.md]:
```suggestion
Run `pulumi new` to create a new project:
```

File: content/docs/tutorials/aws/ecs-fargate.md (hunk 1 above)
Comment (chrsmith) [reply content/docs/tutorials/aws/ecs-fargate.md]: Done.

File: content/docs/tutorials/aws/ecs-fargate.md (hunk 2 above)
Comment (chrsmith) [reply content/docs/tutorials/aws/ecs-fargate.md]: Done.

**Pull request merged**